index.md

```diff
@@ -13,7 +13,7 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
 
 * [CodeSchool: Try Git][]
 * [CodeSchool: Git Real][]
-* [CodeSchool: Git Real 2][] - Levels 1, 2, 4, 6
+* [CodeSchool: Git Real 2][]
```

Comment:
I removed the levels as CodeSchool doesn't allow you to skip lessons. In order to do 4 and 6, you have to have completed 3 and 5.